src/scene/procedural.js

```diff
@@ -639,13 +639,19 @@ pc._createConeData = function (baseRadius, peakRadius, height, heightSegments, c
  */
 pc.createCylinder = function (device, opts) {
     // Check the supplied options and provide defaults for unspecified ones
-    var baseRadius = opts && opts.baseRadius !== undefined ? opts.baseRadius : 0.5;
+    // #ifdef DEBUG
+    if (opts && opts.hasOwnProperty('baseRadius') && !opts.hasOwnProperty('radius'))
+      console.warn('DEPRECATED: "baseRadius" in arguments, use "radius" instead');
```

Comment:
I think that in this PR context, you would change 4 space.
But you revert in following commit, so I questioned. 
https://github.com/playcanvas/engine/pull/973/commits/d0433667d24b72cfc3fea796a36c5948eae61acb